webapp-client/src/App.js

```diff
@@ -0,0 +1,62 @@
+import React, { Fragment } from 'react';
+import logo from './lobster-icon.jpg';
+import './App.css';
+import CategoryList from './component/CategoryList';
+import AddCategoryForm from './component/AddCategoryForm';
+import Category from './component/Category';
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    /** hard-coded for now, will be fetched from db */
+    this.default = 'All';
+    this.state = {
+      /** list of category names for now, will be list of category ids later */
+      categories: [this.default],
+      currentCategory: this.default,
+    };
+    this.setCurrentCategory = this.setCurrentCategory.bind(this);
+    this.addCategory = this.addCategory.bind(this);
+  }
+
+  setCurrentCategory(category) {
+    this.setState({
+      currentCategory: category,
+    });
+  }
+
+  /** Called in AddCategoryForm, to be deleted later when fetching from db */
+  addCategory(category) {
+    this.setState({
+      categories: [...this.state.categories, category],
+    });
+  }
+
+  render() {
+    return (
+      <Fragment>
+        <div className="App">
+          <h1>Lobster Ice Cream</h1>
```

Comment:
bulma title class?

Reply:
We have a navbar PR coming up to address this (#18)